mriqc/interfaces/functional.py

```diff
@@ -309,6 +311,189 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class GatherTimeseriesInputSpec(BaseInterfaceInputSpec):
```

Comment:
`BaseInterfaceInputSpec` hasn't been different from `TraitedSpec` for quite some time. It's still around to keep things from breaking, but we shouldn't use it for new interfaces.

```suggestion
class GatherTimeseriesInputSpec(TraitedSpec):
```